tests/nightly/test_large_array.py

```diff
@@ -1212,6 +1212,15 @@ def test_full():
     assert a[-1][-1] == 3
 
 
+def test_gather():
+    arr = mx.nd.ones((LARGE_X, SMALL_Y))
```

Comment:
We don't need nested parentheses right??

Reply:
We do need parenthesis
```
mx.nd.ones(3,4)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/bapac/workspace/transpose/incubator-mxnet/python/mxnet/ndarray/ndarray.py", line 3153, in ones
    return _internal._ones(shape=shape, ctx=ctx, dtype=dtype, **kwargs)
  File "<string>", line 36, in _ones
  File "/Users/bapac/workspace/transpose/incubator-mxnet/python/mxnet/_ctypes/ndarray.py", line 100, in _imperative_invoke
    ctypes.byref(out_stypes)))
  File "/Users/bapac/workspace/transpose/incubator-mxnet/python/mxnet/base.py", line 254, in check_call
    raise MXNetError(py_str(_LIB.MXGetLastError()))
mxnet.base.MXNetError: [11:15:17] ../include/mxnet/./base.h:526: Invalid context string 4
````